FiveCalls/FiveCalls/Dashboard.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  Dashboard.swift
+//  FiveCalls
+//
+//  Created by Nick O'Neill on 6/28/23.
+//  Copyright © 2023 5calls. All rights reserved.
+//
+
+import SwiftUI
+
+struct Dashboard: View {
+    @EnvironmentObject var store: Store
+
+    @State var showLocationSheet = false
+
+    let op = Operator()
+    
+    var body: some View {
+        NavigationStack {
+            ScrollView {
+                VStack(alignment: .leading, spacing: 10) {
+                    LocationHeader(location: store.state.location, fetchingContacts: store.state.fetchingContacts)
+                        .padding(.bottom, 10)
+                        .onTapGesture {
+                            showLocationSheet.toggle()
+                        }
+                        .sheet(isPresented: $showLocationSheet) {
+                            LocationSheet()
+                                .presentationDetents([.medium])
+                                .presentationDragIndicator(.visible)
+                                .padding(.top, 40)
+                            Spacer()
+                        }
+                    Text("What’s important to you?")
```

Comment:
localize

Reply:
There's a number of missing localizations, added an item to review them all in #331